canonical-web-and-design/ubuntu.com
# Tutorial JS tests

## Done

- Removed example tests
- Add tests for /static/js/src/tutorial.js

## QA

- Check out this feature branch
- Run `./run exec yarn test-js`
- Check that two tests run on the tutorials JS and that they both pass
